src/services/db/GitFileSystemService.ts

```diff
@@ -476,12 +551,46 @@ export default class GitFileSystemService {
       })
   }
 
+  fastForwardMaster(
+    repoName: string
+  ): ResultAsync<boolean, GitFileSystemError> {
+    const efsVolPath = this.getEfsVolPathFromBranch("master")
+    return this.isValidGitRepo(repoName, "master").andThen((isValid) => {
```

Comment:
can I refactor this in a separate PR? tbh I think it's a lot cleaner if all the methods (`isValidGitRepo`, `getFilePathStats`, `isGitInitialized` etc) just take in the correct `efsVolPath`, instead of each one taking in a `isStaging` boolean and then deciding which EFS vol path to use

meaning we just need to `getEfsVolFromBranch` once at the top, and then just pass that to all the random util methods; at the end of the day its just the efs vol that matters

Reply:
sure